scss/components/organisms/_footer.scss

```diff
@@ -0,0 +1,27 @@
+@charset "UTF-8";
+
+//
+// Footer
+//
+// The global footer.
+//
+// Markup: ../../templates/components/organisms/footer/footer.twig
+//
+// Style guide: Organisms.Footer
+//
+
+.footer-wrapper {
+  @include footer-wrapper;
+}
+
+.footer-branding {
+  padding: 0 4rem 0 0;
+}
+
+.footer-menus {
```

Comment:
1. This does not follow BEM naming conventions.
2. Please comment your code describing its purpose.
3. This element should be encapsulated (nested) within the block level element to avoid class scope creep and naming collisions.
4. Why not write css here? Does this really need to be a mixin?